pkg/kubelet/stats/helper.go

```diff
@@ -74,6 +74,13 @@ func cadvisorInfoToContainerStats(name string, info *cadvisorapiv2.ContainerInfo
 		}
 	}
 
+	result.UserDefinedMetrics = cadvisorInfoToUserDefinedMetrics(info)
+
+	// Skip filesystem stats if rootFs and imageFs stats are not passed in.
+	if rootFs == nil || imageFs == nil {
```

Comment:
Maybe make it to be able to handle the case where one of rootFs and imageFs is not nil since it's an utility function?

Something like this?
```
if imageFs != nil {
    result.Rootfs = ...
}
if rootFs != nil {
    // populate the rest
}
```

Reply:
Done.